README.md

```diff
@@ -109,27 +115,30 @@ services:
 Now we run `runserver` manually to have more control over restarts:
 
 ```sh
-docker compose exec django bash
+docker-compose up -d django
+docker-compose exec django bash
 root$ python manage.py runserver 0.0.0.0:8000
 ```
 
 ### Initial Setup
 
+Migrate DB:
+
+    docker-compose run --rm django python manage.py migrate
+
 Create a superuser:
 
     docker-compose run --rm django python manage.py createsuperuser
 
-Migrate DB:
-
-    docker-compose run --rm django python manage.py migrate
+When asked for a username and password, enter values of your choosing.  Email address may be left empty.    
 
 See detailed [cookiecutter-django Docker
 documentation](http://cookiecutter-django.readthedocs.io/en/latest/deployment-with-docker.html).
 
 
 ### Restore database
 
-To restore a database dump, you can run:
+If you have a database dump you wish to restore, you can run:
```

Comment:
Slightly different wording might prevent someone who is quickly running through the readme from dropping the db when they don't have a database dump to use for the restore.